sormas-rest/src/main/java/de/symeda/sormas/rest/ImmunizationResource.java

```diff
@@ -143,4 +146,34 @@ public VaccinationDto createVaccination(
 		return FacadeProvider.getVaccinationFacade().getAllVaccinations(personUuid, disease);
 	}
 
+	@GET
+	@Path("/vaccination/{uuid}")
+	public VaccinationDto getVaccinationByUuid(@PathParam("uuid") String uuid) {
+		return FacadeProvider.getVaccinationFacade().getByUuid(uuid);
+	}
+
+	@POST
+	@Path("/push")
```

Comment:
Please change the path since it creates a conflict with the already existing method `post`.